description: disable third-party cookies by default
author: Andres Salomon <dilinger@debian.org>

This is easily configured in
  Settings -> Security & Privacy -> Cookies & other site data

With this patch, we just change the default on a new chromium profile.


--- a/components/content_settings/core/browser/cookie_settings.cc
+++ b/components/content_settings/core/browser/cookie_settings.cc
@@ -66,7 +66,7 @@ void CookieSettings::RegisterProfilePref
     user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterIntegerPref(
       prefs::kCookieControlsMode,
-      static_cast<int>(CookieControlsMode::kIncognitoOnly),
+      static_cast<int>(CookieControlsMode::kBlockThirdParty),
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 }
 
